packages/scene-composer/stories/useMockedValueDataBindingProvider.ts

```diff
@@ -150,17 +145,23 @@ export function useMockedValueDataBindingProvider(): IValueDataBindingProvider {
                 dataBindingContext: map,
               };
 
-              lastBinding.current = result;
+              lastBinding = result as IValueDataBinding;
 
-              resolve(result);
+              resolve(lastBinding);
             }, MOCK_DELAY);
           });
         },
 
         setOnStateChangedListener(listener: (state: IValueDataBindingProviderState) => void | undefined) {
-          storeRef.current.onStateChangedListener = listener;
+          storeRef.onStateChangedListener = listener;
         },
       };
     },
+    createQuery: (dataBinding: IValueDataBinding) => ({
+      build: () => {
+        throw 'not implemented testing code';
```

Comment:
I recommend throwing an error, instead of a string, whenever possible. Normalizing errors to all be of type `Error` reduces conditional logic downstream and enables consistent handling of errors.

Reply:
sure

Reply:
ok